src/main/java/tds/testpackageconverter/converter/mappers/TestPackageBlueprintMapper.java

```diff
@@ -289,28 +321,38 @@ private static BlueprintElement mapAssessmentBlueprintElements(final Testspecifi
                 })
                 .collect(Collectors.toCollection(LinkedHashSet::new));
 
-        claimBlueprintElements.forEach(blueprintElement -> mapBlueprintElement(blueprintElement, parentIdToBpElement, scoringMap));
+        claimBlueprintElements.forEach(blueprintElement ->
```

Comment:
This isn't a comment for your code, but I wonder if there is a way to do this so you don't have to iterate over the `claimBlueprintElements` twice.  Seems like this for each could be within the flatMap method above which would result in one less collection iteration.